scripts/demo_common.sh

```diff
@@ -128,7 +159,7 @@ trap 'cleanup; trap - HUP; kill -HUP $$' HUP
 trap 'cleanup; trap - INT; kill -INT $$' INT
 trap 'cleanup; trap - TERM; kill -TERM $$' TERM
 
-if config_has MBEDTLS_ENTROPY_NV_SEED; then
+if [ $IS_MBEDTLS_ROOT -eq 1 ] && config_has MBEDTLS_ENTROPY_NV_SEED; then
```

Comment:
I would rather in `config_has` return systematically "do not have" in TF-PSA-Crypto case.